modules/rostests/rosautotest/CConfiguration.cpp

```diff
@@ -75,6 +78,12 @@ CConfiguration::ParseParameters(int argc, wchar_t* argv[])
                     m_Submit = true;
                     break;
 
+                case 't':
+                    ++i;
+                    tmp_RepeatCount = _wtoi(argv[i]);
+                    m_RepeatCount = tmp_RepeatCount == 0 ? 1 : tmp_RepeatCount;
```

Comment:
"== 0" to be replaced by "< 1" in case negative number is used (robustness case)

Reply:
please explain how an unsigned number can be negative.

Reply:
Sorry, I looked at _wtoi rather than tmp_RepeatCount. Disregard.

Reply:
Any need to protect again a max value or do we allow full range of var type ?